### PR TITLE
Add `video` in imagemap

### PR DESCRIPTION
**Pull request by yohsuke**
related issue: #288

* Add supporting play videos in imagemap messages.
* And add sample code in the kitchinsink.

### CODE REVIEW

File: line-bot-model/src/main/java/com/linecorp/bot/model/message/ImagemapMessage.java [hunk 1]
```diff
@@ -65,19 +66,21 @@
...
+    private final ImagemapVideo video;
+
     private final QuickReply quickReply;
 
     /**
-     * Constructor without {@link #quickReply} parameter.
+     * Constructor without {@link #video}, {@link #quickReply} parameter.
      *
-     * <p>If you want use {@link QuickReply}, please use {@link #builder()} instead.
+     * <p>If you want use {@link ImagemapVideo} or {@link QuickReply}, please use {@link #builder()} instead.
```
[the remark this answers is not in the document]
Comment (imasahiro) [reply line-bot-model/src/main/java/com/linecorp/bot/model/message/ImagemapMessage.java]: How about making multiple construcotrs instead?

File: line-bot-model/src/main/java/com/linecorp/bot/model/message/imagemap/ImagemapExternalLink.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+@Value
+public class ImagemapExternalLink {
+    /**
+     * Webpage URL. Called when the label displayed after the video is tapped.
+     *
+     * <ul>
+     * <li>Max: 1000 characters</li>
+     * </ul>
+     */
+    String linkUri;
```
Comment (imasahiro) [inline line-bot-model/src/main/java/com/linecorp/bot/model/message/imagemap/ImagemapExternalLink.java]: Similar to @kazuki-ma 's suggestion at https://github.com/line/line-bot-sdk-java/pull/295#discussion_r236481890
How about making this to `URI`?

File: line-bot-model/src/main/java/com/linecorp/bot/model/message/ImagemapMessage.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (yohsuke) [reply line-bot-model/src/main/java/com/linecorp/bot/model/message/ImagemapMessage.java]: I think it's good ~

File: line-bot-model/src/main/java/com/linecorp/bot/model/message/imagemap/ImagemapExternalLink.java (hunk 2 above)
Comment (yohsuke) [reply line-bot-model/src/main/java/com/linecorp/bot/model/message/imagemap/ImagemapExternalLink.java]: done~

**Comment by yohsuke** [comment]
Thanks~

**Pull request merged**